src/collect.go

```diff
@@ -386,3 +391,16 @@ func getConnectionURLSAP(connectionURL string) string {
 	}
 	return r[len(r)-1]
 }
+
+func getConnectionURLHostPort(connectionURL string) (string, string) {
+	connectionSAP := getConnectionURLSAP(connectionURL)
+	hostParts := strings.Split(connectionSAP, "/")
+	if len(hostParts) == 0 {
+		return "", ""
+	}
+	hostPort := strings.Split(hostParts[0], ":")
```

Comment:
maybe use SplitN(2) or if len(hostPort) < 2
also maybe rename hostPort into hostAndPort or something

Reply:
I would stick to Split() and check if there are exactly 2 items returned. If there more or less it will mean that something is wrong here.